### ISSUE TITLE
returns -1 for fractional values without leading zero

### ISSUE THREAD

**Issue by kylemcdonald** [jakeboone02/numeric-quantity#1]
```js
> require('numeric-quantity')('0.5')
0.5
> require('numeric-quantity')('.5')
-1
```

**Comment by jakeboone02** [comment]
@kylemcdonald, this was sort of on purpose, since my original use case didn't need to handle that scenario.  I'll take a look at it but would be happy to accept a PR if you want to tackle it yourself.

**Comment by kylemcdonald** [comment]
thanks for the clarification @jakeboone02! for now i've switched to `eval(str.replace(' ', '+'))` which isn't as robust as this lib, but solves my case. if i come back to `numeric-quantity` i'll patch and PR.

**Comment by jakeboone02** [comment]
This is fixed with fe56479 and released as 0.2.0 on npm.  Thanks for the nudge, @kylemcdonald!
